docs/guides/setup-channel.mdx

```diff
@@ -274,7 +274,13 @@ Alternatively, you can also access quantum processors with REST APIs, enabling y
       * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
       * If you don't save your credentials, you must specify them every time you start a new session.
 
-1. <span id="cloud-untrusted"></span>**Avoid executing code on an external cloud Python environment to minimize security risks.** If you must use an untrusted environment (on, for example, a public computer), change your API token after each use by deleting it on the [API keys page](https://cloud.ibm.com/iam/apikeys) and creating a new one. To initialize the service in this situation, you can use code like the following:
+1. <span id="cloud-untrusted"></span>**Avoid executing code on an external cloud Python environment to minimize security risks.** If you must use an untrusted environment (on, for example, a public computer), change your API token after each use by deleting it on the [API keys page](https://cloud.ibm.com/iam/apikeys) and creating a new one. 
+
+    <Admonition type="caution">
+      **Protect your API token!** Never include your token in source code, Python script, or notebook file. When sharing code with others, ensure that your API token is not embedded directly within the Python script. Instead, share the script without the token and provide instructions for securely setting it up.
```

Comment:
The Functions guide should be updated to use this wording. Additionally, if you take my recommendation for saying what to do if you accidentally share it, then this file will need that same change.